busted/outputHandlers/junit.lua

```diff
@@ -37,13 +35,17 @@ return function(options)
     return nil, true
   end
 
+  local function formatDuration(duration)
+    return string.format("%.2f", duration)
+  end
+
   local function elapsed(start_time)
-    return string.format("%.2f", (socket.gettime() - start_time))
+    return formatDuration(busted.gettime() - start_time)
   end
 
   handler.suiteEnd = function(suite, count, total)
     local suite = top
-    suite.xml_doc.attr.time = elapsed(suite.start_time)
+    suite.xml_doc.attr.time = formatDuration(suite.duration)
```

Comment:
Line 47 and 48 need to be swapped, the 'top' data structure does not have suite.duration.